src/plugins/index_pattern_field_editor/public/components/flyout_panels/flyout_panels.scss

```diff
@@ -0,0 +1,47 @@
+.fieldEditor__flyoutPanels {
+  height: 100%;
+
+  &__column {
+    height: 100%;
+    overflow: hidden;
+  }
+}
+
+.fieldEditor__flyoutPanel {
+  height: 100%;
+  overflow-y: auto;
+  padding: $euiSizeL;
+
+  &--pageBackground {
+    background-color: $euiPageBackgroundColor;
+  }
+  &--emptyShade {
+    background-color: $euiColorEmptyShade;
+  }
+  &--leftBorder {
+    border-left: $euiBorderThin;
+  }
+  &--rightBorder {
+    border-right: $euiBorderThin;
+  }
+  &--withContent {
+    padding: 0;
+    overflow-y: hidden;
+    display: flex;
+    flex-direction: column;
+  }
+
+  &__header {
+    padding: 0 !important;
+  }
+
+  &__content {
+    flex: 1;
+    overflow-y: auto;
```

Comment:
```suggestion
    overflow-y: auto;
    overflow-x: hidden;
```

This will prevent a horizontal scrollbar from appearing when the autosuggestions list appears.

![image](https://user-images.githubusercontent.com/1238659/129114524-f6218da7-deea-434e-a5f0-58ba1ae98eba.png)